README.md

```diff
@@ -1,4 +1,170 @@
-# pravega-benchmark
-Performance benchmark tool and scripts for Pravega
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
 
-The scripts mentioned here deploys ZK, DL, Pravega and HDFS through scripts on given set of nodes. The perf test runners are also copied to the nodes and benchmarks are measured.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+
+# Pravega Benchmark Tool
+
+The Pravega benchmark tool used for the performance benchmarking of pravega streaming storage cluster.
+This tool performs the throughput and latency analysis for the multi producers/writers and consumers/readers of pravega.
+it also validates the end to end latency. The write and/or read latencies can be stored in a CSV file for later analysis.
+At the end of the performance benchmarking, this tool outputs the 50th, 75th, 95th , 99th and 99.9th latency percentiles.
+
+
+### Prerequisites
+
+- Java 8+
+- Gradle 4+
+
+### Building
+
+Checkout the source code:
+
+```
+git clone https://github.com/pravega/pravega-benchmark
+cd pravega-benchmark
+```
+
+Build the Pravega benchmark Tool:
+
+```
+./gradlew build
+```
+
+untar the Pravega benchmark tool to local folder
+
+```
+tar -xvf ./build/distributions/pravega-benchmark.tar -C ./run
+```
+
+Running Pravega bencmark tool locally:
+
+```
+<dir>/pravega-benchmark$ ./run/pravega-benchmark/bin/pravega-benchmark  -help
+usage: pravega-benchmark
+ -consumers <arg>               number of consumers
+ -controller <arg>              controller URI
+ -events <arg>                  number of events/records if 'time' not
+                                specified;
+                                otherwise, maximum events per second by
+                                producer(s)and/or number of events per
```

Comment:
Missing space after `producer(s)`.

Reply:
yes correcting in readme file and code also.